pkg/kubectl/cmd/util/helpers.go

```diff
@@ -476,6 +480,10 @@ func DumpReaderToFile(reader io.Reader, filename string) error {
 	return nil
 }
 
+func GetServerSideApplyFlag(cmd *cobra.Command) bool {
+	return GetFlagBool(cmd, "serverside-apply")
```

Comment:
Should this just be `serverside`? Not super important since we don't expect people to play with that flag so much, but this looks weird:
```
kubectl apply --serverside-apply ...
```